transcoder/src/extract.go

```diff
@@ -68,16 +112,26 @@ func extractSubs(info *MediaInfo) error {
 				cmd.Args,
 				"-map", fmt.Sprintf("0:s:%d", *sub.Index),
 				"-c:s", "copy",
-				fmt.Sprintf("%s/%d.%s", subs_path, *sub.Index, *ext),
+				fmt.Sprintf("%s/%d.%s", subtitlesWorkingDirectory, *sub.Index, *ext),
 			)
 		}
 	}
 	log.Printf("Starting extraction with the command: %s", cmd)
 	cmd.Stdout = nil
-	err := cmd.Run()
-	if err != nil {
+
+	if err := cmd.Run(); err != nil {
 		fmt.Println("Error starting ffmpeg extract:", err)
 		return err
 	}
+
+	// Move attachments and subtitles to the storage backend
+	// TODO should existing files be removed?
```

Comment:
the `defer os.RemoveAll(workdir)` should already handle that.

Reply:
I was more referring to the files in the storage backend. However, given the small file sizes, this is probably fine to leave until I implement some sort of cleanup for transcoded files. WDYT?

Reply:
I think it's fine to never delete those files, we currently never delete them